qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -1063,6 +1063,195 @@ def GaussianSquareDrag(
     return instance
 
 
+def GaussianSquareEcho(
+    duration: Union[int, ParameterExpression],
+    amp: Union[float, ParameterExpression],
+    sigma: Union[float, ParameterExpression],
+    width: Optional[Union[float, ParameterExpression]] = None,
+    angle: Optional[Union[float, ParameterExpression]] = 0.0,
+    active_amp: Optional[Union[float, ParameterExpression]] = 0,
+    active_angle: Optional[Union[float, ParameterExpression]] = 0,
+    risefall_sigma_ratio: Optional[Union[float, ParameterExpression]] = None,
+    name: Optional[str] = None,
+    limit_amplitude: Optional[bool] = None,
+) -> SymbolicPulse:
+    """An echoed Gaussian square pulse with an active tone overlaid on it.
```

Comment:
```suggestion
    """An echoed Gaussian square pulse with an active tone overlaid on it.
    
```

Nit pick but first paragraph appears in the autosummary so this cannot be very long.